frontend_tests/node_tests/markdown.js

```diff
@@ -47,19 +49,18 @@ var doc = "";
 set_global('document', doc);
 
 set_global('$', function (obj) {
-  if (typeof obj === 'function') {
-    // Run on-load setup
-    obj();
-  } else if (typeof obj === 'string') {
-    // $(document).on usage
-    // Selector usage
-    return {on: function () {}};
-  }
+    if (typeof obj === 'function') {
```

Comment:
Do we need this still? It seems like we're just stubbing jQuery here, so you might want to change this with like `global.make_zjquery()` or something along those lines.